Lib/test/test_sqlite3/test_userfunctions.py

```diff
@@ -421,6 +421,26 @@ def test_func_return_illegal_value(self):
         self.assertRaisesRegex(sqlite.OperationalError, msg,
                                self.con.execute, "select badreturn()")
 
+    def test_func_keyword_args(self):
+        regex = (
+            r"Passing keyword arguments 'name', 'narg' and 'func' to "
+            r"_sqlite3.Connection.create_function\(\) is deprecated. "
+            r"Parameters 'name', 'narg' and 'func' will become "
+            r"positional-only in Python 3.15."
+        )
+
+        def check(*args, **kwds):
+            with self.assertWarnsRegex(DeprecationWarning, regex) as cm:
+                self.con.create_function(*args, **kwds)
+            self.assertEqual(cm.filename, __file__)
```

Comment:
Checking only `__file__` is not enough because there are two frames with the same filename. It would be simpler to inline this function. It is just 3 lines.

Reply:
Good point; I'll inline it.